### PR TITLE
Dev chumpol v4

### PR DESCRIPTION
**Pull request by OctoPol7**

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,20 +1,66 @@
...
+    'SofiaPro-Black': require('./assets/fonts/SofiaPro-Black.otf'),
+    'SofiaPro-Bold': require('./assets/fonts/SofiaPro-Bold.otf'),
+    'SofiaPro-Light': require('./assets/fonts/SofiaPro-Light.otf'),
+    'SofiaPro-Medium': require('./assets/fonts/SofiaPro-Medium.otf'),
+    'SofiaPro-Regular': require('./assets/fonts/SofiaPro-Regular.otf'),
+    'SofiaPro-SemiBold': require('./assets/fonts/SofiaPro-SemiBold.otf'),
+    Quicksand: require('./assets/fonts/Quicksand-Bold.ttf'),
+  });
+
+  console.log(fontsLoaded);
```
Comment (SVerma799) [inline App.js]: Remove the extra logs

Comment (OctoPol7) [reply App.js]: already removed in other commits

File: screens/User/Profile.js [hunk 2]
```diff
@@ -27,7 +27,11 @@ export default function Profile({ navigation }) {
           <Text fontWeight='semibold'>{currentUser.displayName}</Text>
           <Text>{currentUser.readerType}</Text>
           <Text fontStyle='italic' fontSize='sm'>
-            joined {new Date(currentUser.createdAt).toISOString().substring(0, 10).replaceAll('-', '/')}
+            joined{' '}
```
Comment (SVerma799) [inline screens/User/Profile.js:31]: This thing usually comes up with formatter. Be carefull next time

File: App.js (hunk 1 above)
Comment (SVerma799) [reply App.js]: I checked it now. Apologies for that

File: screens/User/Profile.js (hunk 2 above)
Comment (SVerma799) [reply screens/User/Profile.js:31]: What It mean is that to add a space in UI and I think you can add the space normally.

File: App.js [hunk 3]
```diff
@@ -1,20 +1,66 @@
...
+    if (fontsLoaded) {
+      await SplashScreen.hideAsync();
+    }
+  }, [fontsLoaded]);
+
+  if (!fontsLoaded) {
+    return null;
+  }
+
+  const theme = extendTheme({
```
Comment (SVerma799) [inline App.js:37]: Please remind to fix this on friday. Just a small improvement what we can do is we can keep App.js file clear just have imports in here and move them to Another file and export them. We can try to do it...

**Pull request merged**